content/sensu-go/6.8/operations/deploy-sensu/secure-postgres.md

```diff
@@ -0,0 +1,374 @@
+**NOTE**: This guide describes one option for securing communication between Sensu and PostgreSQL and is intended as a starting point.
+Your organization's needs may require a different approach.
+{{% /notice  %}}
+
+## Prerequisites
+
+To use this guide, you must have:
+
+- A running Sensu deployment.
+- A running PostgreSQL instance.
+
+## Install cfssl
+
+The [CloudFlare cfssl][2] toolkit is released as a collection of command-line tools.
+
+If you followed [Generate certificates for your Sensu installation][3], you already downloaded and installed the Cloudflare cfssl toolkit.
+If not, run the following commands:
+
+{{< language-toggle >}}
+
+{{< code "RHEL/CentOS" >}}
+sudo curl -s -L -o /bin/cfssl https://github.com/cloudflare/cfssl/releases/download/v1.6.2/cfssl-bundle_1.6.2_linux_amd64
+
+sudo curl -s -L -o /bin/cfssl-certinfo https://github.com/cloudflare/cfssl/releases/download/v1.6.2/cfssl-certinfo_1.6.2_linux_amd64
+
+sudo chmod +x /bin/cfssl*
+{{< /code >}}
+
+{{< code "Ubuntu/Debian" >}}
+
+# Update apt repos
+sudo apt-get update
+
+# Install cfssl
+sudo apt-get install golang-cfssl
+
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+To verify that cfssl is installed, run:
+
+{{< code shell >}}
+cfssl version
+{{< /code >}}
+
+## Create a Certificate Authority (CA)
+
+Follow these steps to create a CA with cfssl and cfssljson:
+
+1. Create `/etc/sensu/tls` (which does not exist by default):
+{{< code shell >}}
+mkdir -p /etc/sensu/tls
+{{< /code >}}
+
+2. Navigate to the new /etc/sensu/tls directory:
+{{< code shell >}}
+cd /etc/sensu/tls
+{{< /code >}}
+
+3. Create the CA:
+{{< code shell >}}
+echo '{"CN":"Sensu Test CA","key":{"algo":"rsa","size":2048}}' | cfssl gencert -initca - | cfssljson -bare ca -
+{{< /code >}}
+
+4. Define signing parameters and profiles:
+{{< code shell >}}
+echo '{"signing":{"default":{"expiry":"17520h","usages":["signing","key encipherment","client auth"]},"profiles":{"postgresql":{"usages":["signing","key encipherment","server auth","client auth"],"expiry":"4320h"},"backend":{"usages":["signing","key encipherment","client auth"],"expiry":"4320h"}}}}' > ca-config.json
+{{< /code >}}
+{{% notice note %}}
+**NOTE**: We suggest a 6-month expiry duration for security, but you can use any duration you prefer when you define the `expiry` attribute value in the signing parameters.
+{{% /notice %}}
+
+<a id="copy-ca-pem-postgres"></a>
+
+You should now have a directory at `/etc/sensu/tls` that contains the following files:
+
+ filename        | description |
+-----------------|-------------|
+`ca.pem`         | CA root certificate. Required for all systems running the Sensu backend or agent. The agent and backend use `ca.pem` to validate server certificates at connection time. |
+`ca-key.pem`     | CA root certificate private key. |
+`ca-config.json` | CA signing parameters and profiles. Not used by Sensu. |
+`ca.csr`         | Certificate signing request for the CA root certificate. Not used by Sensu. |
+
```

Comment:
At this point, I only have `ca-config.json` in my /etc/sensu/tls -- maybe because cfssl isn't installing correctly?